cpp/dolfinx/fem/assemble_matrix_impl.h

```diff
@@ -163,6 +163,7 @@ void assemble_cells(
 /// matrix.
 /// @param x_dofmap Dofmap for the mesh geometry.
 /// @param x Mesh geometry (coordinates).
+/// @param num_cell_facets Number of cell facets
```

Comment:
Should we call this `num_facets_per_cell`, this would be more consistent with what we use elsewhere:
https://github.com/search?q=repo%3AFEniCS%2Fdolfinx%20_per_&type=code